src/modules/services/hooks/useServiceDialog.tsx

```diff
@@ -190,6 +190,7 @@ export function useServiceDialog({
           )}
           {selectedService && formDefinition && serviceType && (
             <DynamicForm
+              clientId={enrollment?.client.id}
```

Comment:
The service form doesn't need this, right? Since it's only used for the "autofill" pickers? Just wondering.

Reply:
I guess not, probably not needed on the referral either. I'll pull it out of those components. Thanks!